src/features/domain-settings/Steps/DetailsStep.module.scss

```diff
@@ -0,0 +1,93 @@
+@import '../../../../node_modules/@zero-tech/zui/styles/_colors.scss';
+@import '../../../../node_modules/@zero-tech/zui/styles/_typography.scss';
+
+.FormContainer {
+	display: flex;
+	flex-direction: column;
+
+	.ScrollSection {
+		display: flex;
+		flex-direction: column;
+		max-height: 55vh;
+		gap: 1.5rem;
+		overflow-y: auto;
+		padding-right: 0.5rem;
+
+		&[data-variant='locked'] {
+			opacity: 0.5;
+			overflow-y: hidden;
+		}
+
+		.DetailsContainer {
+			display: flex;
+			flex-direction: row;
+			gap: 1.5rem;
+
+			.InputWrapper {
+				display: flex;
+				flex-direction: column;
+				gap: 1.5rem;
+				width: 100%;
+
+				.Input {
+					gap: 0.25rem;
+
+					> *:last-child {
+						width: unset;
+					}
+				}
+			}
+		}
+
+		.SwitchSettingsContainer {
+			border-top: 2px solid $grey-lighter-2;
+
+			h5 {
+				margin: 0.5rem 0;
+				line-height: 1.5rem;
+			}
+
+			.SwitchGroup {
+				display: flex;
+				flex-direction: column;
+				padding: 0;
+
+				.SwitchRow {
+					display: flex;
+					flex-direction: row;
+					padding: 1rem 0;
+					border-bottom: 1px solid $mine-shaft;
+				}
+
+				.SwitchInput {
+					gap: 0.25rem;
+
+					> *:last-child {
+						width: unset;
+					}
+				}
+			}
+		}
+
+		// Scrollbar
+		&::-webkit-scrollbar-corner {
+			background: $black;
+			height: 0;
+			width: 0;
+		}
+
+		&::-webkit-scrollbar-track {
+			background-color: none;
+		}
+
+		&::-webkit-scrollbar {
+			width: 0.25rem;
+			background-color: none;
+		}
+
+		&::-webkit-scrollbar-thumb {
+			border-radius: 0.625rem;
+			background-color: rgba(60, 161, 255, 0.2);
+		}
```

Comment:
Hmm maybe we need a utility component for this.

Reply:
What would be the best way to create a utility component for use with `module.scss` files?